backend/controllers/studyzone.controller.js

```
@@ -31,18 +48,21 @@ export const updateSession = async (req, res) => {
       return res.status(400).json({ message: 'Invalid session ID' });
     }
 
-    const sessionDate = date ? new Date(date) : undefined;
-    if (sessionDate) sessionDate.setHours(0, 0, 0, 0); // Normalize time to midnight
+    if (!isValidDuration(duration)) {
+      return res.status(400).json({ message: 'Invalid duration' });
+    }
+
+    let updateObj = { duration };
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'updateObj' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)